testdata/config/scylla.yaml

```diff
@@ -8,3 +8,4 @@ client_encryption_options:
   keyfile: /etc/scylla/db.key
   truststore: /etc/scylla/ca.crt
   require_client_auth: true
+force_schema_commit_log: true
```

Comment:
Why do we need to explicitly mention this param?
What about other params that are set by default in the Scylla.yaml?
Are we overriding all of them using this yaml?

Reply:
Without this the container fails to start. There is an issue about that when using 5.4.x we have to specify that option: https://github.com/scylladb/scylladb/issues/16427#issuecomment-1874165269

Reply:
@sylwiaszunejko , can I ask you to add comment to the file so that we could link this option to the context